docs/user/faqs.rst

```diff
@@ -61,3 +61,34 @@ testing on 14.04 or older.
 
 .. _see changelog: https://poppler.freedesktop.org/releases.html
 .. _install manually: https://poppler.freedesktop.org
+
+How can I use use Fonduer for documents in Languages other than English?
+--------------------------------------------------------------------------------
+
+If available, Fonduer uses languages supported by spaCy for tokenization and 
+its NLP pipeline (`see spacy language support`_).
+We also started adding languages with spaCy alpha support for tokenization
+(`see spacy alpha languages`_). 
+
+If you would like to use Fonduer for Japanese documents,
+you will first have to install some additional packages (`see mecab on PyPI`_).
+
+For Linux::
+
+    $ sudo apt-get install libmecab-dev
+    $ sudo apt-get install mecab mecab-ipadic-utf8
+
+For OSX::
+
+    $ brew install mecab
+    $ brew install mecab-ipadic
+
+Afterwards, you can use the ``make dev_extra`` command to install Fonduer with
```

Comment:
Only developers should be dealing with `make`ing the dev environment. Since this doc is for end users, they should be doing something like 

```
pip install fonduer[spacy_ja]
```

I'd also make it explicit in the following sentence that Japanese is the _only_ extra language integrated in Fonduer.